app/src/main/java/com/example/whatTravel/Activities/NearbySearchActivity.java

```diff
@@ -79,15 +76,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     public void onDataSuccessLoad(int loaderId, String response) {
-        if(loaderId == WTAPIConstants.LOAD_NEARBY_PARK) {
+        if(loaderId == WTAPIConstants.LOAD_NEARBY_PARK || loaderId ==LOAD_MORE) {
            NearbySearchResults nearbySearchResults = ResponseParser.getNearbySearchResults(response);
-           WTLog.debug("parser....",nearbySearchResults.toString());
            if (nearbySearchResults !=null) {
               String status = nearbySearchResults.getStatus();
               if ("OK".equalsIgnoreCase(status)) {
+                  token = nearbySearchResults.getNextToken();
```

Comment:
Yes it will, this token is used for loadMore, If this token is null, the loadMore will not be launched